rules/linux/process_creation/proc_creation_lnx_password_group_discovery.yml

```
@@ -0,0 +1,27 @@
+title: Detection of Password Groups Discovery
```

```suggestion
title: Potential Linux Groups Discovery
```
Potential  -> https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq-title-convention.md
I can't find anything better to match the 3 selection.